README.md

```diff
@@ -219,6 +226,8 @@ then here's the best choice:
 
 Run `make duck_s3_ls_then_cloudfront`
 
+**Note:** You need a valid AWS credentials to run this command.
```

Comment:
I found that I would get an empty `CC-MAIN-2024-22.files` when I didn't have valid credentials.